packages/geo/VALIDATION.md

```diff
@@ -0,0 +1,192 @@
+# External Validation Against geographiclib-mgrs
+
+This document summarizes the validation of the `@accelint/geo` package against the [geographiclib-mgrs](https://github.com/spatialillusions/geographiclib-mgrs) library, which is a widely-used JavaScript port of GeographicLib C++ by Charles Karney.
```

Comment:
"widely-used" feels like a stretch for a lib with 4 stars on github and less than 700 weekly downloads on [npm](https://www.npmjs.com/package/geographiclib-mgrs). maybe i'm misreading and widely-used is referring to the c++ lib?

do we feel we can trust the js port?